_docs_operate/modules.md

```diff
@@ -87,6 +87,11 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### Sever sent events <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The server sent events module allows the connector to open an one way connection from the backbone to the connector.
```

Comment:
Capitalize the words "Module", "Connector" and "Backbone" as it is already done in the rest of the text on this page. Same for the sentence below.